ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactionsTest.java

```diff
@@ -690,9 +724,12 @@ public void assertThatCorrectNonceIsReturned() {
         .isPresent()
         .hasValue(6);
     addLocalTransactions(6, 10);
```

Comment:
We have a logic change here. Previous implementation intended for oldest tx (1) regardless of nonce to be dropped.  Since nonce distance is considered before sequence (and it must, because sequences are global) in the new logic, nonce 6 gets dropped. Next nonce is expected to be 6, and i do believe that is the correct next nonce to return, however, in reality 6 is gone.